lib/vagrant-openshift/action/generate_template.rb

```diff
@@ -122,8 +122,9 @@ def find_ami_from_tag(box_info)
 
             aws_compute = Fog::Compute.new(fog_config)
             @env[:ui].info("Searching for latest base AMI")
-            images = aws_compute.images.all({'Owner' => 'self', 'name' => "#{box_info[:aws][:ami_tag_prefix]}*",
-                                             'state' => 'available' })
+            image_filter = {'Owner' => 'self', 'name' => "#{box_info[:aws][:ami_tag_prefix]}*", 'state' => 'available' }
+            image_filter.merge! @options[:required_tags] if @options[:required_tags]
```

Comment:
I'm thinking ahead; this allows matching any tag including custom tags.
Just in case
On Apr 19, 2016 6:11 PM, "Dan McPherson" <notifications@github.com> wrote:

> In lib/vagrant-openshift/action/generate_template.rb
> <https://github.com/openshift/vagrant-openshift/pull/436#discussion_r60321313>
> :
>
> > @@ -122,8 +122,9 @@ def find_ami_from_tag(box_info)
> >
> >              aws_compute = Fog::Compute.new(fog_config)
> >              @env[:ui].info("Searching for latest base AMI")
> > -            images = aws_compute.images.all({'Owner' => 'self', 'name' => "#{box_info[:aws][:ami_tag_prefix]}*",
> > -                                             'state' => 'available' })
> > +            image_filter = {'Owner' => 'self', 'name' => "#{box_info[:aws][:ami_tag_prefix]}*", 'state' => 'available' }
> > +            image_filter.merge! @options[:required_tags] if @options[:required_tags]
>
> Don't you just want to match the "Name" tag
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/vagrant-openshift/pull/436/files/e61b8a9be846e2f80bd10e2a3ddcba68529867ac#r60321313>
>